### PR TITLE
Delegation Documentation Update to 1.19.1

### PR DESCRIPTION
**Pull request by DamjanOstrelic**

### CODE REVIEW

**Review by oldcryptogeek — APPROVED** [review]

**Review by oldcryptogeek — COMMENTED** [review]
LGTM

**Review by Scitz0 — APPROVED** [review]
LGTM, didn't go through every single command but skimmed through it all and seems good.

File: docs/Staking/Delegates.md [hunk 1]
```diff
@@ -1,268 +1,302 @@
...
+# Generate the stake address key-pair
+cardano-cli shelley stake-address key-gen \
+--verification-key-file stake.vkey \
+--signing-key-file stake.skey
+
+# Generate the actual stake address with the stake key-pair
+cardano-cli shelley stake-address build \
+--stake-verification-key-file stake.vkey \
+--out-file stake.addr \
+--mainnet
```
Comment (rdlrt) [inline docs/Staking/Delegates.md:32]: Perhaps we can add a comment to say use `--mainnet` or `--testnet-magic 1097911063`

**Review by rdlrt — APPROVED** [review]
Possibly just need to add a comment for testnet-magic usage (for the first operation) , LGTM otherwise 

Thanks

File: docs/Staking/Delegates.md (hunk 1 above)
Comment (DamjanOstrelic) [reply docs/Staking/Delegates.md:32]: I'll add a note near the first --mainnet call

**Pull request merged**